cypress/integration/frontend-testing.js

```diff
@@ -1,23 +1,119 @@
-function runFrontendTest(name) {
-    it(name, () => {
+import {it} from 'mocha';
+import {assertNoConsoleOutput, stubConsoleOutput} from '../support/utils';
+
+const PaneDataMap = {
```

Comment:
small nit; case style
```suggestion
const PANE_DATA_MAP = {
```